docs/man-openmpi/man3/MPI_Status_get_error.3.rst

```diff
@@ -0,0 +1,71 @@
+.. _mpi_status_get_error:
+
+
+MPI_Status_get_error
+========================
+
+.. include_body
+
+:ref:`MPI_Status_get_error` |mdash| Retrieves the MPI_ERROR field from *status*.
+
+
+SYNTAX
+------
+
+
+C Syntax
+^^^^^^^^
+
+.. code-block:: c
+
+   #include <mpi.h>
+
+   int MPI_Status_get_error(MPI_Status *status, int *error)
+
+
+Fortran Syntax
+^^^^^^^^^^^^^^
+
+.. code-block:: fortran
+
+   USE MPI
+   ! or the older form: INCLUDE 'mpif.h'
+   MPI_STATUS_GET_ERROR(STATUS, ERROR, IERROR)
+   	INTEGER	STATUS(MPI_STATUS_SIZE), IERROR
+   	INTEGER ERROR
+
+
+Fortran 2008 Syntax
+^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: fortran
+
+   USE mpi_f08
+   MPI_Status_get_error(status, error, ierror)
+   	TYPE(MPI_Status), INTENT(INOUT) :: status
+   	INTEGER, INTENT(OUT) :: error
+   	INTEGER, OPTIONAL, INTENT(OUT) :: ierror
+
+
+INPUT/OUTPUT PARAMETER
+----------------------
+* ``status``: Status with which to retrieve error (status).
+
+INPUT PARAMETER
```

Comment:
```suggestion
OUTPUT PARAMETER
```

Reply:
I don't think you just want to make this 1-line change -- I think that all 3 params should be merged into a single "OUTPUT" section.

Reply:
(I deleted my previous comment)

Why do you suggest this?  Looking at MPI-4.1 p42:11, `status` is an IN parameter.

Reply:
mistake I guess.